neurom/view/common.py

```diff
@@ -26,20 +26,9 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 # SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-"""
-Module containing the common functionality
-to be used by view-plot modules.
-"""
-from neurom import NeuriteType
+"""Functionality for styling plots"""
```

Comment:
Most of the changes in this file have to do w/:
* remove kwargs when possible, make all args explicit: better for completion, better for error handling, easier to write docstrings for